### DISCUSSION TITLE
feat: Negative Prompts and Prefilled Beginning of Response

### DISCUSSION THREAD

**Discussion by notasquid1938** [open-webui/open-webui#16045] [Ideas]
Adding input fields for a negative prompt for the beginning of the model's response can increase fine control over output.

**Comment by jsuelwald** [comment]
I'd like to re-open this issue - or at least make that availible as a discussion, because this (and other parameters for A1111) would be nice to have configured.

**Reply by rgaricano** [reply]
I think that there are a specific disscusion about negative prompt for comfyui /a1111,
I understand that @notasquid1938 request for a negative prompt for model, but this can be done in system prompt, as also can be done in comfyui/a1111 prompt, or so it seems to me.
e.g. add to system prompt,
```
negative prompt, avoid:
- xxxxxx
- yyyyyy
```
